src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml

```diff
@@ -1,44 +1,46 @@
-		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.XpathInjectionPlugin</ascanrule>                
-		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.XXEPlugin</ascanrule>
-	</ascanrules>
-	<pscanrules/>
-	<filters/>
-	<files/>
-	<not-before-version>2.4.0</not-before-version>
-	<not-from-version/>
-</zapaddon>
+<zapaddon>
+	<name>Active scanner rules (beta)</name>
+	<version>17</version>
+	<description>The beta quality Active Scanner rules</description>
+	<author>ZAP Dev Team</author>
+	<url/>
+	<changes>
+	<![CDATA[
+	Moved Format String scanner from alpha to beta.<br>
+    ]]>
+	</changes>
+	<extensions>
+		<extension>org.zaproxy.zap.extension.ascanrulesBeta.ExtensionAscanRulesBeta</extension>
+	</extensions>
+	<ascanrules>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.BackupFileDisclosure</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.BufferOverflow</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.CrossDomainScanner</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.Csrftokenscan</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.ExpressionLanguageInjectionPlugin</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.FormatString</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.HeartBleedActiveScanner</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.HPP</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.InsecureHTTPMethod</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.LDAPInjection</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.PaddingOraclePlugin</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.RemoteCodeExecutionCVE20121823</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SessionFixation</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.ShellShockScanner</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SQLInjectionHypersonic</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SQLInjectionMySQL</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SQLInjectionOracle</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SQLInjectionPostgresql</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SourceCodeDisclosureCVE20121823</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SourceCodeDisclosureSVN</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.SourceCodeDisclosureWEBINF</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.UsernameEnumeration</ascanrule>
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.XpathInjectionPlugin</ascanrule>                
+		<ascanrule>org.zaproxy.zap.extension.ascanrulesBeta.XXEPlugin</ascanrule>
+	</ascanrules>
+	<pscanrules/>
+	<filters/>
+	<files/>
+	<not-before-version>2.4.0</not-before-version>

```

Comment:
Noticed now, this has to be changed to 2.4.1 because the scanner is using new features (method ```targets(TechSet)```).